app/_hub/kong-inc/upstream-oauth/overview/_index.md

```diff
@@ -0,0 +1,76 @@
+---
+nav_title: Overview
+title: Overview
+---
+
+The Upstream OAuth plugin allows {{site.base_gateway}} to support OAuth flows between Kong and the upstream API.
+
+The plugin supports storing tokens issued by the IdP in different backend formats.
+
+## How it works
+
+The upstream OAuth2 credential flow works similarly to the [client credentials grant](/hub/kong-inc/openid-connect/how-to/authentication/client-credentials/) used by the OpenID Connect plugin. If a cached access token isn't found, Kong issues a request to the IdP token endpoint to obtain a new token, which is cached, and then passed to the upstream API via a configurable header (`Authorization` by default).
+
+<!--vale off-->
+
+{% mermaid %}
+sequenceDiagram
+    autonumber
+    participant client as Client <br>(e.g. mobile app)
+    participant kong as API Gateway <br>(Kong)
+    participant idp as IDP <br>(e.g. Keycloak)
+    participant api as 3rd Party API
+    activate client
+    activate kong
+    client->>kong: request to Kong
+    deactivate client
+    activate idp
+    kong->>idp: request access token <br>from IdP using <br>client ID and client secret (if IdP auth is set)
+    deactivate kong
+    idp->>idp: authenticate client
+    activate kong
+    idp->>kong: return access token
+    deactivate idp
+    activate api
+    kong->>api: request with access token in <br>Authorization header
```

Comment:
```suggestion
    kong->>api: request with access token in <br><br>Authorization header
```
I don't know if this will work but on this screen it looks like this: 
![image](https://github.com/user-attachments/assets/62ef9e61-ebc6-4ec2-a998-97f0e5e84d76)